packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -864,6 +864,14 @@ async function runScript() {
         default: return undefined;
       }
     })
+    .option('inputBranded', {
+      boolean: true,
+      default: false,
+    })
+    .option('outputBranded', {
+      boolean: true,
+      default: false,
+    })
```

Comment:
Changing everything else to use `option` seems like a good way to handle this.

Reply:
Fixed it.

Since the order was not exactly aligned with declaration order in the code and I don't know how exactly they were decided, 
I adjusted declaration order a bit to preserve the original printed order.